mean-module/index.js

```diff
@@ -0,0 +1,190 @@
+      }, {
+        value: 'addServerPoliciesFolder',
+        name: 'policies',
+        checked: false
+      }, {
+        value: 'addServerRoutesFolder',
+        name: 'routes',
+        checked: true
+      }, {
+        value: 'addServerTestsFolder',
+        name: 'tests',
+        checked: true
+      }]
+    }];
+
+    this.prompt(prompts, function (props) {
+      // name
+      this.name = props.name;
+
+      // client-side
+      this.addConfigFolder = props.addConfigFolder;
+      this.addControllersFolder = props.addControllersFolder;
+      this.addCSSFolder = props.addCSSFolder;
+      this.addDirectivesFolder = props.addDirectivesFolder;
+      this.addFiltersFolder = props.addFiltersFolder;
+      this.addImagesFolder = props.addImagesFolder;
+      this.addServicesFolder = props.addServicesFolder;
+      this.addTestsFolder = props.addTestsFolder;
+      this.addViewsFolder = props.addViewsFolder;
+
+      //server-side
+      this.addServerConfigFolder = props.addServerConfigFolder;
+      this.addServerControllersFolder = props.addServerControllersFolder;
+      this.addServerConfigFolder = props.addServerConfigFolder;
+      this.addServerPoliciesFolder = props.addServerPoliciesFolder;
+      this.addServerRoutesFolder = props.addServerRoutesFolder;
+      this.addServerTestsFolder = props.addServerTestsFolder;
+
+      done();
+    }.bind(this));
+  },
+
+  renderModule: function () {
+    this.slugifiedName = s(this.name).humanize().slugify().value();
+    this.humanizedName = s(this.slugifiedName).humanize().value();
+
+    // Create module folder
+    mkdirp.sync('modules/' + this.slugifiedName + '/client/');
+
+    // Create client module sub-folders
+    if (this.addConfigFolder) {
+      mkdirp.sync('modules/' + this.slugifiedName + '/client/config');
+    }
+    if (this.addControllersFolder) {
+      mkdirp.sync('modules/' + this.slugifiedName + '/client/controllers');
+    }
+    if (this.addCSSFolder) {
+      mkdirp.sync('modules/' + this.slugifiedName + '/client/css');
+    }
+    if (this.addDirectivesFolder) {
+      mkdirp.sync('modules/' + this.slugifiedName + '/client/directives');
+    }
+    if (this.addFiltersFolder) {
+      mkdirp.sync('modules/' + this.slugifiedName + '/client/filters');
+    }
+    if (this.addImagesFolder) {
+      mkdirp.sync('modules/' + this.slugifiedName + '/client/img');
+    }
+    if (this.addServicesFolder) {
+      mkdirp.sync('modules/' + this.slugifiedName + '/client/services');
+    }
+    if (this.addViewsFolder) {
+      mkdirp.sync('modules/' + this.slugifiedName + '/client/views');
+    }
+
+    if (this.addTestsFolder) {
+      mkdirp.sync('modules/' + this.slugifiedName + '/tests/client');
+    }
+
+    // Render angular module definition
+    this.template('_.client.module.js', 'modules/' + this.slugifiedName + '/client/' + this.slugifiedName + '.client.module.js');
+
+    // Create server module sub-folders
+    if (this.addServerConfigFolder) {
```

Comment:
lines 165 to 179 all create the same folder, so can't you just do 1 if statement by or'ing all the conditions together?